### PR TITLE
Improved Orchestrator Pricing

### PR DESCRIPTION
**Pull request by EAsuperstar**

### CODE REVIEW

File: ai/pipelines/text-to-image.mdx [hunk 1]
```diff
@@ -157,6 +157,18 @@ The following system requirements are recommended for optimal performance:
...
+
+<Note>We are planning to simplify the pricing in the future so orchestrators can set 
+one AI price per compute unit and have the system automatically scale based on the 
+model's compute requirements.</Note>
+
+The pricing for the `text-to-image` pipeline is based on competitor pricing. However, 
+we strongly encourage orchestrators to set their own pricing based on their costs and 
+requirements. Setting a competitive price will help attract more jobs, as Gateways can 
+set their maximum price for a job. The currently recommended pricing for this pipeline 
+is `1.9073484e-08 USD` per **output pixel**(`height * width * images`).
```
Comment (rickstaa) [inline ai/pipelines/text-to-image.mdx:170]:
```suggestion
is `1.9073484e-08 USD` per **output pixel** (`height * width * images`).
```

**Review by rickstaa — APPROVED** [review]
LGTM thanks!

File: ai/pipelines/audio-to-text.mdx [hunk 2]
```diff
@@ -107,6 +107,18 @@ The following system requirements are recommended for optimal performance:
...
+
+<Note>We are planning to simplify the pricing in the future so orchestrators can set 
+one AI price per compute unit and have the system automatically scale based on the 
+model's compute requirements.</Note>
+
+The pricing for the `audio-to-text` pipeline is based on competitor pricing. However, 
+we strongly encourage orchestrators to set their own pricing based on their costs and 
+requirements. Setting a competitive price will help attract more jobs, as Gateways can 
+set their maximum price for a job. The currently recommended pricing for this pipeline 
+is `0.02e-6 USD` per **milliseconds** of input.
```
Comment (rickstaa) [inline ai/pipelines/audio-to-text.mdx:120]:
```suggestion
is `0.02e-6 USD` per **milliseconds** of audio input.
```

File: ai/pipelines/audio-to-text.mdx [hunk 3]
```diff
@@ -107,6 +107,18 @@ The following system requirements are recommended for optimal performance:
...
+
+<Note>We are planning to simplify the pricing in the future so orchestrators can set 
+one AI price per compute unit and have the system automatically scale based on the 
+model's compute requirements.</Note>
+
+The pricing for the `audio-to-text` pipeline is based on competitor pricing. However, 
+we strongly encourage orchestrators to set their own pricing based on their costs and 
+requirements. Setting a competitive price will help attract more jobs, as Gateways can 
+set their maximum price for a job. The currently recommended pricing for this pipeline 
+is `0.02e-6 USD` per **milliseconds** of audio input.
```
Comment (rickstaa) [inline ai/pipelines/audio-to-text.mdx:120]:
```suggestion
is `0.02e-6 USD` per **milliseconds** of input audio.
```

File: ai/pipelines/audio-to-text.mdx (hunk 2 above)
Comment (ecmulli) [inline ai/pipelines/audio-to-text.mdx]: Grammar is a bit off. 3 options:

```The recommended pricing for this pipeline 
is currently `0.02e-6 USD` per **milliseconds** of input.
```

```The recommended pricing for this pipeline 
is `0.02e-6 USD` per **milliseconds** of input.
```

```The current recommended pricing for this pipeline 
is `0.02e-6 USD` per **milliseconds** of input.
```

Comment (EAsuperstar) [reply ai/pipelines/audio-to-text.mdx]: Good catch, thanks. Will make the corrections.

**Pull request merged**